providers/fcontext.rb

```diff
@@ -25,6 +25,23 @@ def restorecon(file_spec)
   return "restorecon -iR #{path}"
 end
 
+def semanage_options(filetype)
+  # Set options for filetype
+  if filetype
```

Comment:
```
irb(main):003:0> puts 'a' if ""
(irb):3: warning: string literal in condition
a
```
We'll need to do something like `unless filetype.nil? or filetype.empty?`
Also do all platforms support `a`?